src/middlewared/middlewared/plugins/zettarepl.py

```diff
@@ -862,12 +862,17 @@ async def _define_ssh_transport(self, credentials):
         except CallError as e:
             raise CallError(f"Error while querying SSH key pair for credentials {credentials['id']}: {e!s}")
 
+        cipher = credentials["attributes"]["cipher"].lower()
+        if (await self.middleware.call("system.security.config"))["enable_fips"]:
+            cipher = "fips"
+
         return {
             "hostname": credentials["attributes"]["host"],
             "port": credentials["attributes"]["port"],
             "username": credentials["attributes"]["username"],
             "private-key": key_pair["attributes"]["private_key"],
             "host-key": credentials["attributes"]["remote_host_key"],
+            "cipher": cipher,
```

Comment:
@yocalebo we never used this option value (by accident I think) and no one has complained. `none` cipher is still unsupported in SCALE and no one has ever tried to use it. Should we just remove this option altogether?

Reply:
Yeah, just remove it.